cli/src/replace_node/deploy-replacement

```diff
@@ -162,6 +162,7 @@ function system_states {
         # "system.storage.multipath.install"
         # "system.storage.multipath.config"
         "system.network"
+        "system.network.mgmt.service_port"
```

Comment:
@ypise  looks like not related thing, do we have a confidence here?

Reply:
Yes, it a required feature.
We are not performing boxing-unboxing after a node replacement. Hence, we need this.

It's also present: https://github.com/Seagate/cortx-prvsnr/blob/f1151526312d2689b7f625b8b4b507ebebd13eac/cli/src/factory_ops/boxing/prov_tasks#L115
```
function configure_service_port {
    salt "srvnode-1" state.apply components.system.network.mgmt.service_port ${salt_opts} || (
        echo "ERROR: Failed to configure service port on srvnode-1." | tee -a ${LOG_FILE}
        exit 25
    )
}
```